packages/aws-cdk-lib/cloud-assembly-schema/lib/assets/docker-image-asset.ts

```diff
@@ -63,6 +63,15 @@ export interface DockerImageSource {
    */
   readonly dockerBuildArgs?: { [name: string]: string };
 
+  /**
+   * SSH agent socket or keys
+   *
+   * Requires building with docker buildkit.
+   *
+   * @default - No ssh flag is set
+   */
+  readonly dockerBuildSSH?: string;
```

Comment:
```suggestion
  readonly dockerBuildSsh?: string;
```

I know the PR needs clean-up anyway, but let's go with this casing everywhere please!

Reply:
Changed!